src/service/fixed-layer.js

```diff
@@ -187,7 +190,9 @@ export class FixedLayer {
           ownerNode.hasAttribute('amp-extension')) {
         continue;
       }
-      this.discoverSelectors_(cssRules);
+      // Don't dereference cssRules we need to avoid "Can't access rules"
+      // DOMException due to reading a CORS stylesheet e.g. font.
+      this.discoverSelectors_(stylesheet.cssRules);
```

Comment:
Should we add a try-catch?

Reply:
Searched error reporter for "cssRules" and only saw one report from amp-animation.